evap/staff/templates/staff_user_merge_selection.html

```diff
@@ -8,20 +8,50 @@
 
 {% block content %}
     {{ block.super }}
-    <h3>{% trans 'Merge users' %}</h3>
-
-    <form id="user-selection-form" method="POST" class="form-horizontal">
-        {% csrf_token %}
-        <div class="card mb-3">
-            <div class="card-body">
-                <p>{% trans 'Select the users you want to merge.' %}</p>
-                {% include 'bootstrap_form.html' with form=form wide=True %}
+    <div class="row row-cols-1 row-cols-md-2">
+        <div class="col">
+            <div class="card">
+                <div class="card-body">
+                    <h4 class="card-title">{% trans 'Merge users' %}</h4>
+                    <form id="user-selection-form" method="POST" class="form-horizontal">
+                        {% csrf_token %}
+                        <p>{% trans 'Select the users you want to merge.' %}</p>
+                        {% include 'bootstrap_form.html' with form=form wide=True %}
+                    </form>
+                </div>
+                <div class="card-footer text-center card-submit-area d-flex">
+                    <button type="submit" form="user-selection-form" class="btn btn-light mx-auto my-auto">{% trans 'Show merge info' %}</button>
+                </div>
             </div>
         </div>
-        <div class="card card-submit-area text-center mb-3">
-            <div class="card-body">
-                <button type="submit" class="btn btn-primary">{% trans 'Show merge info' %}</button>
+        <div class="col">
+            <div class="card">
+                <div class="card-body">
+                    <h4 class="card-title">{% trans 'Merge suggestions' %}</h4>
+                    <table id="merge-suggestion-table" class="table table-striped table-narrow table-vertically-aligned">
```

Comment:
The `id` seems to be unused here.

```suggestion
                    <table class="table table-striped table-narrow table-vertically-aligned">
```